plugins/available/functional-test-plugin/functionalTestPlugin.js

```diff
@@ -118,6 +138,39 @@ class FunctionalTestPlugin {
     await this.sdk.server.info();
   }
 
+  // accessors.registerSubscription related methods ============================
+
+  async registerSubscription(request) {
+    const roomId = await this.context.accessors.realtime.subscribe(
```

Comment:
I must be missing something in the code :thinking: 
I see that you define `accessors.subscription.(un)register` methods in the plugin context, but here you seem to call `accessors.realtime.(un)subscribe` without error :thinking: 

What's going on?